pyresample/geometry.py

```diff
@@ -769,13 +769,17 @@ def __init__(self, area_id=None, description=None, projection=None,
         self.area_id = area_id
         self.description = description
         self.proj_dict = proj_dict
-        self.width = self.width = width
-        self.height = self.height = height
+        self.width = width
+        self.height = height
         self.area_extent = area_extent
         self.optimize_projection = optimize_projection
         self.resolution = resolution
         self.rotation = rotation
 
+    @property
+    def shape(self):
+        return self.height, self.width
```

Comment:
Considering that height and width could be None I'm not sure I'm comfortable with this. I'll let @mraspaud weigh in though.

Reply:
Yeah, I can change it to make shape None; I just added a description, but I just want consistency in properties between DynamicAreaDefinition and AreaDefinition

Reply:
I would rather not having the property at all, although consistency is nice.